### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 7

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#183] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252610-5639a88f-65a8-42ea-b851-73b078d8dd96.png)

**Comment by Exyss** [comment]
$\binom{16}{8} \binom{8}{8}$

_Stesso approccio risolutivo usato per gli esercizi #11, #12_
